c/misra/src/rules/RULE-16-5/DefaultNotFirstOrLastOfSwitch.ql

```diff
@@ -0,0 +1,23 @@
+/**
+ * @id c/misra/default-not-first-or-last-of-switch
+ * @name RULE-16-5: A default label shall appear as either the first or the last switch label of a switch statement
+ * @description Locating the default label is easy when it the first of last label.
+ * @kind problem
+ * @precision very-high
+ * @problem.severity error
```

Comment:
I suggest lowering this to `recommendation` because having the `default` label in a different location doesn't result in a bug.
```suggestion
 * @problem.severity recommendation
```